test/test_authn.py

```diff
@@ -77,8 +77,6 @@ def client(make_client):
 def test_authn_signin_good_credentials(client):
     resp = client.post(URL_SIGNIN, json=CREDENTIALS_0)
     assert resp.status_code == 200
-    assert resp.json["code"] == "ok"
-    assert resp.json["msg"] == "Successful login"
```

Comment:
We could put the test of the message back in:

```
    assert resp.json["msg"] == "Successful login"
```

If we do not want the old message then at least we should test that the message is the empty string (as it is now in this pull request):

```
assert resp.json["msg"] == ""
```